nix/ci.nix

```diff
@@ -6,5 +6,4 @@ let
 in pkgs.mkShell {
     nativeBuildInputs = ourpkgs.native-deps;
     buildInputs = ourpkgs.build-deps;
-    LD_LIBRARY_PATH = "${pkgs.z3.lib}/lib";
```

Comment:
Explicitly setting `LD_LIBRARY_PATH` isn't necessary because this is already dealt with by setting `nativeBuildInputs`. Note I used `nativeBuildInputs` because that's what usually works for me but I'm not sure this is the exact right `*buildInputs` attribute that should be used as [the documentation](https://nixos.org/manual/nixpkgs/stable/#variables-specifying-dependencies) is quite technical.